website/server/libs/setupPassport.js

```diff
@@ -2,6 +2,70 @@ import passport from 'passport';
 import nconf from 'nconf';
 import { Strategy as FacebookStrategy } from 'passport-facebook';
 import { Strategy as GoogleStrategy } from 'passport-google-oauth20';
+import _ from 'lodash';
+import validator from 'validator';
+import * as passwordUtils from './password';
+import { model as User } from '../models/user';
+import { BasicStrategy } from 'passport-http';
+import { Strategy as LocalStrategy } from 'passport-local';
+import { Strategy as ClientPasswordStrategy } from 'passport-oauth2-client-password';
+import { Strategy as BearerStrategy } from 'passport-http-bearer';
+import {
+  NotAuthorized
+} from './errors';
+
+/**
+ * LocalStrategy
+ *
+ * This strategy is used to authenticate users based on a username and password.
+ * Anytime a request is made to authorize an application, we must ensure that
+ * a user is logged in before asking them to approve the request.
+ */
+passport.use(new LocalStrategy(
```

Comment:
Otherwise the verification code for username and passwords should be abstracted somewhere so it's not duplicated